### PR TITLE
D8 nid 410

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: nidirect_gp/nidirect_gp.module [hunk 1]
```diff
@@ -130,6 +149,33 @@ function nidirect_gp_entity_presave(EntityInterface $entity) {
...
 }
 
+/**
+ * Implements hook_views_pre_render().
+ */
+function nidirect_gp_views_pre_render(ViewExecutable $view) {
+
+  if ($view->id() == 'gp_practices' && $view->current_display == 'find_a_gp') {
+    // Display singular or plural result count.
+    $result_count = Drupal::translation()->formatPlural(
```
Comment (johangant) [inline nidirect_gp/nidirect_gp.module]: Should this be `\Drupal::translation()` (leading backslash)?

Comment (omahm) [reply nidirect_gp/nidirect_gp.module]: Does work without but for standards I'll add it in.

File: nidirect_gp/nidirect_gp.module [hunk 2]
```diff
@@ -87,6 +105,14 @@ function nidirect_gp_form_alter(&$form, FormStateInterface $form_state, $form_id
       t('You cannot edit the title. It is created automatically from the practice name and surgery name.');
     $form['title']['#disabled'] = 'TRUE';
 
+    // Hide the Prescriptions URL title as we will use a default.
+    $form['field_gp_prescriptions']['widget']['0']['title']['#type'] = 'hidden';
+    $form['field_gp_prescriptions']['widget']['0']['title']['#default_value'] = 'Repeat prescriptions';
```
Comment (johangant) [inline nidirect_gp/nidirect_gp.module]: `t('Repeat prescriptions');` ?

File: nidirect_gp/nidirect_gp.module [hunk 3]
```diff
@@ -87,6 +105,14 @@ function nidirect_gp_form_alter(&$form, FormStateInterface $form_state, $form_id
...
+
+    // Hide the Appointments URL title as we will use a default.
+    $form['field_gp_appointments']['widget']['0']['title']['#type'] = 'hidden';
+    $form['field_gp_appointments']['widget']['0']['title']['#default_value'] = 'Online appointments';
```
Comment (johangant) [inline nidirect_gp/nidirect_gp.module]: `t('Online appointments');` ?

**Pull request merged**